class11.1/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <vector>
+#include <cstdlib>
+#include <ctime>
+
+void proposeBook(std::vector<std::string>& books) {
+    if (books.empty()) {
+        std::cout << "Adios amigo!" << std::endl;
+        exit(0);
```

Comment:
it is better to avoid such calls of termination in function except main. In general decision of program exit should be done on higher level of abstraction except maybe some fatal errors.
Therefore you should return status instead of exit. And the result should be handled by client to make a decision.
WHy it is important? with explicit exit inside function you will not be able to write unit tests for this function